### PR TITLE
Use Docker to Build DEB & RPM packages

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

**Comment by portertech** [comment]
Docker doesn't currently support  i386 :-1:

**Comment by portertech** [comment]
Ready for review :+1: 

pauly-phone

**Comment by agoddard** [comment]
docker docker docker docker docker docker docker docker docker docker docker docker docker docker docker docker docker

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+image: bradrydzewski/base
+cache:
+  - /var/run
```
Comment (portertech) [inline .drone.yml]: This is a HACK to work around the lack of direct Docker volumes support. This allows the container to use the host's Docker.

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'mixlib-shellout', '1.4.0'
```
Comment (portertech) [inline Gemfile]: This is so build containers can use Ruby < 1.9.3

**Comment by agoddard** [comment]
:boom:

**Pull request merged**